Core/Config.h

```diff
@@ -394,6 +394,8 @@ struct Config {
 	bool bEnableAdhocServer;
 	int iWlanAdhocChannel;
 	bool bWlanPowerSave;
+	bool bEnableNetworkChat;
+	int iNewChat;
```

Comment:
What is `iNewChat` and why is it part of the ppsspp config struct?  It looks like it may be for counting how many new chat messages there are, but then it probably doesn't belong here.

-[Unknown]